### PR TITLE
Add gefs inline post itag file

### LINKED ISSUE #1059: Add itag file for GEFS inline post
Add a itag file for GEFS inline post.

### LINKED ISSUE #1061: Update gfs inline post itag files
To approach the plan in which all auxiliary files for gfs/gefs inline post are used from UPP repository, an update is needed for the gfs  itag files.

### PR DESCRIPTION
**Pull request by WenMeng-NOAA**
Main changes include:

- Add gefs inline post itag file
- Update gfs inline post itag files

### CODE REVIEW

File: parm/gefs/post_itag_gefs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+&MODEL_INPUTS
```
Comment (aerorahul) [inline parm/gefs/post_itag_gefs]: can you name this the same way as gfs; e.g. `post_itag_gefs128`?

Comment (WenMeng-NOAA) [reply parm/gefs/post_itag_gefs]: @aerorahul Updated.

**Review by aerorahul — APPROVED** [review]
looks good.

**Comment by WenMeng-NOAA** [comment]
@gspetro-NOAA @FernandoAndrade-NOAA You may start the UPP RTs on R&D machines. There should be no baseline changes with this PR.

**Comment by WenMeng-NOAA** [comment]
The UPP RTs have been completed on WCOSS2 without baseline changes.

**Comment by WenMeng-NOAA** [comment]
@gspetro-NOAA Thanks for testing on R&D machines. Please go ahead to approve this PR.

**Review by gspetro-NOAA — APPROVED** [review]
The RTs passed on Hera, Orion, and Hercules with no baseline change.

**Comment by WenMeng-NOAA** [comment]
This PR is ready for merging.

**Pull request merged**